app/components/election-time-input.js

```diff
@@ -0,0 +1,58 @@
+// https://github.com/EnCiv/undebate-ssp/issues/6
+
+import { useEffect, useRef, useState } from "react"
+import { createUseStyles } from "react-jss"
+import svg from "../../assets/svg/clock.svg"
+
+const ElectionTimeInput = props => {
+    const { defaultValue = "", onDone = () => {} } = props
```

Comment:
I'm going to add a new convention.  Please accept className and style in props, and use them on the outer element of your code.  (in this case the input). 

To combine className passed and the className you want to add - 
```
import cx from 'classnames'
   ...
const {className, style, ... } = props
<input className={cx(className, classes.electionTimeInput)} style={style}
```
This way the parent element could apply formatting this component.

I have done this in some elements in unpoll.  I was thinking about it here because a parent might not want to force the width to 100% - I just depends on how it's going to be used.

Reply:
This makes sense, I noticed this pattern in some unpoll code. I'll implement it.